### ISSUE TITLE
UnsupportedMessageType : $AIALR

### ISSUE THREAD

**Issue by icemagno** [tbsalling/aismessages#53]
I'm receiving the error ```UnsupportedMessageType: $AIALR``` when trying this code:

```
NMEAMessageHandler aisMessageReceiver = new NMEAMessageHandler("TEST", new MessageConsumer() );
NMEAMessage message = NMEAMessage.fromString( "$AIALR,185615.00,025,A,V,AIS: external EPFS lost*23");
aisMessageReceiver.accept( message );
```

The trace:

```
$AIALR,185615.00,025,A,V,AIS: external EPFS lost*23
dk.tbsalling.aismessages.nmea.exceptions.UnsupportedMessageType: $AIALR
	at dk.tbsalling.aismessages.nmea.messages.NMEAMessage.validate(NMEAMessage.java:134)
	at dk.tbsalling.aismessages.nmea.messages.NMEAMessage.<init>(NMEAMessage.java:124)
	at dk.tbsalling.aismessages.nmea.messages.NMEAMessage.fromString(NMEAMessage.java:47)
```

**Comment by icemagno** [comment]
Ok.. I think I've found my error. Seems I missunderstood the application of ```AISMessage``` and ```NMEAMessage```. I'm using some examples from [here](https://github.com/tbsalling/aismessages/blob/master/src/main/java/dk/tbsalling/aismessages/demo/SimpleDemoApp.java) and [this code](https://github.com/tbsalling/aismessages/issues/10) (an UDPInputStream prototype).

Can you clarify me that difference and how can I convert from one to other if possible?

**Comment by icemagno** [comment]
Thanks for this.
Actually I'm reading data from COM port so I have to feed an OutputStream to send it to an InputStream and it is very hard to do because I start to work with AIS few days ago ( a new client ).

I spent about a week just to find you ( other libs are outdated ) and another week just to place myself in the right way. Another week just to think how to feed data without reading from a file or a string (the only examples I found here ).

It was hard to find out how to constantly take strings one by one from the COM port and feed them into the InputStream needed by AISMessages lib. 

I have 5 threads working until now because a heavy blocking rebelion :-)

But I think that UDPInputStream solved my problem ( yes .. hard to tell but I must send and receive UDP packets inside my own code to avoid a lot of blocking Streaming transfer ... 

Don't worry ... I'll learn how to do it in a better fashion.
